specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -471,6 +471,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
+            "description": "Optional, used to get soft deleted volume groups only if set to true. If it is not set for a soft deleted volume groups, it will return 404. If set for active volume groups, it will return 404.",
```

Comment:
Please note, this parameter in this place is for list operation, so I don't think it will return 404. 
If nothing to list out, it should just return 0 items.

Please describe the difference behavior of list API with/without this parameter. Like : with this header only soft deleted volumn groups will be returned.